src/scenes/CollectionGalleryPage/CollectionGalleryPage.tsx

```diff
@@ -32,6 +34,29 @@ function CollectionGalleryPage({ collectionId, username, queryRef }: CollectionG
     });
   }, [username, collectionId, track]);
 
+  const ePress = useKeyDown('e');
+  const escapePress = useKeyDown('Escape');
+  const backspacePress = useKeyDown('Backspace');
+
+  const { push } = useRouter();
+
+  const navigateToEdit = useCallback(() => {
+    void push(`/edit?collectionId=${collectionId}`);
+  }, [push, collectionId]);
+  const navigateToUserGallery = useCallback(() => {
+    void push(`/${username}`);
+  }, [push, username]);
+
+  useEffect(() => {
+    if (ePress) {
+      navigateToEdit();
+    }
+    // FIXME: Do we also want the escape/backspace keypress to trigger navigation to /{username}? Not in spec
+    if (escapePress || backspacePress) {
+      navigateToUserGallery();
+    }
```

Comment:
Flagging

Reply:
let's do it!

Reply:
i think we can toss `backspacePress` though. just escape is fine